cloudinit/distros/networking.py

```diff
@@ -175,6 +176,9 @@ def wait_for_physdevs(
         if strict:
             raise RuntimeError(msg)
 
+    def try_set_link_up(self, devname: DeviceName) -> bool:
```

Comment:
As we expect this to have an implementation in all `Networking` subclasses, we should mark it as an abstractmethod. `is_physical` in this class is an example of how to do this; https://docs.python.org/3/library/abc.html is the stdlib documentation for abstract base classes.